Production/python/TauL2Prod.py

```diff
@@ -0,0 +1,185 @@
+    FailPath = cms.untracked.vstring(),
+    IgnoreCompletely = cms.untracked.vstring(),
+    Rethrow = cms.untracked.vstring(),
+    SkipEvent = cms.untracked.vstring(),
+    allowUnscheduled = cms.obsolete.untracked.bool,
+    canDeleteEarly = cms.untracked.vstring(),
+    emptyRunLumiMode = cms.obsolete.untracked.string,
+    eventSetup = cms.untracked.PSet(
+        forceNumberOfConcurrentIOVs = cms.untracked.PSet(
+            allowAnyLabel_=cms.required.untracked.uint32
+        ),
+        numberOfConcurrentIOVs = cms.untracked.uint32(1)
+    ),
+    fileMode = cms.untracked.string('FULLMERGE'),
+    forceEventSetupCacheClearOnNewRun = cms.untracked.bool(False),
+    makeTriggerResults = cms.obsolete.untracked.bool,
+    numberOfConcurrentLuminosityBlocks = cms.untracked.uint32(1),
+    numberOfConcurrentRuns = cms.untracked.uint32(1),
+    numberOfStreams = cms.untracked.uint32(0),
+    numberOfThreads = cms.untracked.uint32(1),
+    printDependencies = cms.untracked.bool(False),
+    sizeOfStackForThreadsInKB = cms.optional.untracked.uint32,
+    throwIfIllegalParameter = cms.untracked.bool(True),
+    wantSummary = cms.untracked.bool(False)
+)
+
+# Production Info
+process.configurationMetadata = cms.untracked.PSet(
+    annotation = cms.untracked.string('tau_hlt nevts:-1'),
+    name = cms.untracked.string('Applications'),
+    version = cms.untracked.string('$Revision: 1.19 $')
+)
+
+# Output definition
+if isData:
+    process.RECOSIMoutput = cms.OutputModule("PoolOutputModule",
+        dataset = cms.untracked.PSet(
+            dataTier = cms.untracked.string(''),
+            filterName = cms.untracked.string('')
+        ),
+        fileName = cms.untracked.string('tau_hlt_HLT.root'),
+        outputCommands = process.RECOSIMEventContent.outputCommands,
+        splitLevel = cms.untracked.int32(0)
+    )
+
+# Additional output definition
+
+# Other statements
+from HLTrigger.Configuration.CustomConfigs import ProcessName
+process = ProcessName(process)
+
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run3_mc_GRun', '')
+
+# Path and EndPath definitions
+process.endjob_step = cms.EndPath(process.endOfProcess)
+if isData:
+    process.RECOSIMoutput_step = cms.EndPath(process.RECOSIMoutput)
+
+
+# Schedule definition
+process.schedule = cms.Schedule()
+process.schedule.extend(process.HLTSchedule)
+if isData:
+    process.schedule.extend([process.endjob_step,process.RECOSIMoutput_step])
+else:
+    process.schedule.extend([process.endjob_step])
+from PhysicsTools.PatAlgos.tools.helpers import associatePatAlgosToolsTask
+associatePatAlgosToolsTask(process)
+
+# customisation of the process.
+
+
+# Automatic addition of the customisation function
+
+if isData:
+    # Customisation from command line
+    from HLTrigger.Configuration.customizeHLTforCMSSW import customisePixelGainForRun2Input,synchronizeHCALHLTofflineRun3on2018data
+    def customiseFor2018Input(process):
```

Comment:
if you define it as a function, you should call it just below. Perhaps it is better to make two calls directly without defining this function?